ntbs-service/Models/NotificationBannerModel.cs

```diff
@@ -30,7 +28,7 @@ public class NotificationBannerModel
         public string Source;
         public NotificationStatus NotificationStatus;
         public string NotificationStatusString;
-        public bool ShowLink = false;
+        public bool ShowLink;

```

Comment:
I recall the `ShowLink` and `ShowPadlock` functionality being a little convoluted so not sure if this could exhibit some unintended behaviour.
I assume you suitably checked places this was set/used, but thought I would bring some specific attention to it.

Reply:
It was just a redundant assignment, as the `false` is the default value, but happy to put it back in if you think it makes the logic clearer?

Reply:
I think it does make the logic slightly more clear, but it does feel weird to be explicit in default behaviour for that... but nothing else - yeah maybe stripping it out is best :)